ee/fe/modules/500-openvpn/hooks/easyrsa_migration.go

```diff
@@ -95,6 +95,8 @@ func applyMigrationSecretFilter(obj *unstructured.Unstructured) (go_hook.FilterR
 }
 
 func migration(input *go_hook.HookInput) error {
+	input.ConfigValues.Remove("openvpn.storageClass")
```

Comment:
Could you please add a comment on why should we delete this field in the easyrsa migration?

Reply:
Done